ape_etherscan/verify.py

```diff
@@ -248,6 +248,8 @@ def attempt_verification(self):
         }
 
         evm_version = compiler_used.settings.get("evmVersion")
+        license_code = self.license_code
+        license_code_value = None if not license_code else license_code.value
```

Comment:
This reads a bit confusingly at first, what about:

```suggestion
        license_code_value = license_code.value if license_code else None
```